cmake/projects/CURL/hunter.cmake

```diff
@@ -98,6 +98,21 @@ hunter_add_version(
     c15efe3c6c77c5f5d5b098b73d31cc3fbbc0d2fc
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    CURL
+    VERSION
+    "7.74.0-p0"
+#    URL
+#    "https://github.com/cpp-pm/curl/archive/v7.74.0-p0.tar.gz"
+#    SHA1
+#    9166518326aaf2bd5b6b425cf981b18cdde0f9e0
+    URL
+    "https://github.com/ned14/curl/archive/hunter-7.74.0.tar.gz"
```

Comment:
Do you want to revert this back to the cpp-pm version @ned14?

Reply:
I guess so. The only thing I tried removing was ruslo's intsizeof replacement, but it made no difference to CI. So assuming he had good reason to do the replacement in the first place, I guess we keep it.

Reply:
Can you remove the custom version then? 😄